incubator/java-microprofile/templates/default/pom.xml

```diff
@@ -102,7 +102,6 @@
                     <serverName>defaultServer</serverName>
```

Comment:
Maybe there's a case to be explicit sometimes even when using default values for these config parms... but I'd particularly question the **packageFile** and **include** parms... are we just copying these over or did we review them?

Reply:
Just copied them over. Did not review them